.circleci/config.yml

```diff
@@ -5,13 +5,25 @@ jobs:
     steps:
       - checkout
       - run: ./.circleci/setup-rngd.sh
+      - run: ./docker/build-server-base.sh
```

Comment:
Please consider encapsulating the `build` and `deploy` scripts already inside the relevant folders (`client`,`e2e`, `projects` (which should be renamed to `server` very soon)

Reply:
It's a concern that we can solve a bit later. I agree with you generally that it should be done.